src/state/reducers/search.js

```diff
@@ -0,0 +1,44 @@
+import ActionTypes from '../actions/action-types';
+
+/**
+ * searchReducer
+ */
+export const searchesReducer = (state = {}, action) => {
+  switch (action.type) {
+    case ActionTypes.REQUEST_SEARCH:
+      return {
+        ...state,
+        [action.targetId]: {
+          [action.searchId]: {
```

Comment:
We probably want to include `...state[action.targetId]` in all these reducers, right (for either when I have multiple search companion windows open, or two windows with the same manifest)?

Reply:
I'm not sure I follow here. Maybe we can chat quickly?

Reply:
👍 added